fdbclient/ServerKnobs.cpp

```diff
@@ -1410,4 +1410,6 @@ void ServerKnobs::initialize(Randomize randomize, ClientKnobs* clientKnobs, IsSi
 		    (5.0 * VERSIONS_PER_SECOND);
 		clientKnobs->INIT_MID_SHARD_BYTES = MIN_SHARD_BYTES;
 	}
+
+	init(BULK_LOAD_USE_SST_INGEST, true); // Enable SST ingestion by default
```

Comment:
Can you do `if (isSimulated) BULK_LOAD_USE_SST_INGEST = deterministicRandom()->coinflip()` to improve the test coverage?